src/tests/ftest/launch.py

```diff
@@ -956,13 +957,13 @@ def archive_files(destination, host_list, source_files):
     #
     # pylint: disable=bad-continuation
     commands = [
-        "set -eu",
+        "set -ux",
         "rc=0",
         "copied=()",
-        "for file in $(ls {})".format(source_files),
+        "for file in $(ls -d {})".format(source_files),
         "do ls -sh $file",
-        "/lib/daos/TESTING/ftest/cart/util/cart_logtest.py $file",
-        "if scp $file {}:{}/${{file##*/}}-$(hostname -s)".format(
+        "/lib/daos/TESTING/ftest/cart/cart_logtest.py $file",
```

Comment:
We should not be using fixed paths here.  The launch.py command needs to be compatible with running out of a manual build environment and DAOS RPM install environment.   Expanding the current directory should get the correct path for the file:
```suggestion
        "{} $file".format(
            os.path.join(os.path.abspath("cart"), "cart_logtest.py")),
```

Reply:
👍  Thank you!